Tests/MessagingInApp/MessagingInAppTest.swift

```diff
@@ -66,36 +49,18 @@ class MessagingInAppTest: UnitTest {
 
         assertModuleInitialized(isInitialized: false, givenEventListener: nil)
     }
-
-    // MARK: initialize functions with SDK initialized
-
-    func test_initialize_givenSdkInitialized_expectModuleIsInitialized() {
-        MessagingInApp.initialize(siteId: .random, region: .US)
-
-        assertModuleInitialized(isInitialized: true, givenEventListener: nil)
-    }
-
-    // MARK: initialize functions with SDK not initialized
-
-    func test_initialize_givenSdkNotInitialized_expectModuleIsInitialized() {
-        sdkInitializedUtilMock.underlyingIsInitlaized = false
-
-        MessagingInApp.initialize(siteId: .random, region: .US)
-
-        assertModuleInitialized(isInitialized: true, givenEventListener: nil)
-    }
```

Comment:
Because modules are now independent of SDK initialization, we don't need these tests at the moment.